### DISCUSSION TITLE
Undecoded ir signal

### DISCUSSION THREAD

**Discussion by BagiDe** [bengtmartensson/IrScrutinizer#416] [Decoding issues] [answered]
Hi there,
I have provider's digital cable set-top box. Type: Vodafone TV.
When I want to capture it in the Parametric Remote section but I get an error message: Not decodeable signal.
What might be the problem? 
I attached the number 1 button.

[Vodafone TV.zip](https://github.com/bengtmartensson/IrScrutinizer/files/5875140/Vodafone.TV.zip)

**Comment by bengtmartensson** [comment, marked as answer]
Non decoded signal are a fact of life; not a bug. It is just a signal that we cannot recognize, with the current state of knowledge. Therefore, I have moved it to "discussions".  In IrScrutinizer, you can capture with  raw remote instead.

If you want to, you can capture all the signals of your remote, upload, and we will try to analyze them.

    $ irptransmogrifier analyze 0000 0072 0000 0012 000B 000B 0005 000B 0005 001C 0005 000B 0005 000B 0005 000B 0005 000B 0005 000B 0005 000B 0005 000B 0005 0014 0005 0014 0005 0014 0005 000B 0005 000B 0005 000B 0005 0014 0005 094F      
    {36.4k,42,msb}<3,-7|3,-14>(7,-7,A:16,3,-65m)*{A=0x4071}

**Comment by BagiDe** [comment]
OK.
I'm a newbie in ir signals.
Thank you.

Bengt Martensson <notifications@github.com> ezt írta (időpont: 2021. jan.
27., Sze 9:24):
